webrtc.html

```diff
@@ -4450,6 +4450,19 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
         </li>
       </ul>
     </section>
+
+    <section>
+      <h3>RTCDataChannel and Workers</h3>
+      <p>An <code><a>RTCDataChannel</a></code> object MUST be
+      <code><a href="https://html.spec.whatwg.org/multipage/infrastructure.html#transferable-objects">
+      Transferable</a></code> to <code>Workers</code> [[!WEBWORKERS]].  When it's
```

Comment:
The specification is confusing on this point:
http://www.w3.org/TR/html51/infrastructure.html#transferable-objects
"Some objects support being copied and closed in one operation. This is called transferring the object, and is used in particular to transfer ownership of unsharable or expensive resources across worker boundaries."

@annevk can you weigh in here?